howtos/tech/build-cmocka.rst

```diff
@@ -0,0 +1,135 @@
+.. _build-cmocka:
+
+Build Cmocka for Xtensa
+#######################
+
+Cmocka for SOF is built automatically by default.
+However sometimes you may need to have prebuilt version that can be used with :ref:`CMOCKA_DIRECTORY <cmocka-directory-label>`.
```

Comment:
Lines 6-7: suggest "... automatically by default, however you may need a prebuilt version ... "